PepperDashEssentials/Bridges/GlsOccupancySensorBaseControllerBridge.cs

```diff
@@ -31,13 +31,24 @@ public static void LinkToApi(this GlsOccupancySensorBaseController occController
 
             #region Single and Dual Sensor Stuff
             occController.IsOnline.LinkInputSig(trilist.BooleanInput[joinMap.IsOnline]);
+            trilist.StringInput[joinMap.Name].StringValue = occController.Name;
+
+            trilist.OnlineStatusChange += new Crestron.SimplSharpPro.OnlineStatusChangeEventHandler((d, args) =>

```

Comment:
for future reference, this line can be just `trilist.OnlineStatusChange += (d, args) => ...`. The `new Crestron.SimplSharPro.OnlineStatusChangeEventHandler` is not required